### DISCUSSION TITLE
[@types/nodemailer]: Missing `keepBcc` property in `MimeNode`

### DISCUSSION THREAD

**Discussion by AnandChowdhary** [DefinitelyTyped/DefinitelyTyped#69409] [Issues with a @types package]
Additional context is available in https://github.com/nodemailer/nodemailer/issues/1342:

![CleanShot 2024-04-22 at 09 16 54@2x](https://github.com/nodemailer/nodemailer/assets/2841780/e70f09ca-21eb-4eb3-acad-d60a56dea6d5)

[a comment by an automated account is not shown]

**Comment by ayan-bloomscorp** [comment]
@rogierschouten @dex4er @bioball Do you have any plans for this? I know it may not be a top priority, but not having access to `keepBcc` while using the types is a bit frustrating.

**Comment by sagezoic** [comment]
@rogierschouten @dex4er @bioball Hi, can we create a issue for it, i will pick it up and resolve it?

**Comment by MysteryBlokHed** [comment]
Is this still an issue? There seems to be `keepBcc` in the latest versions of the type definitions:

<https://github.com/DefinitelyTyped/DefinitelyTyped/blob/34b8a7244b96bdd38db672ffd94b3a8066eeb868/types/nodemailer/lib/mime-node/index.d.ts#L40-L41>

<https://github.com/DefinitelyTyped/DefinitelyTyped/blob/34b8a7244b96bdd38db672ffd94b3a8066eeb868/types/nodemailer/v6/lib/mime-node/index.d.ts#L40-L41>
